Formula/jenkins-lts.rb

```diff
@@ -6,13 +6,19 @@ class JenkinsLts < Formula
 
   bottle :unneeded
 
-  depends_on :java => "1.8"
+  depends_on "openjdk@11"
 
   def install
-    system "jar", "xvf", "jenkins.war"
+    system Formula["openjdk@11"].opt_bin "/jar", "xvf", "jenkins.war"
     libexec.install "jenkins.war", "WEB-INF/lib/cli-#{version}.jar"
-    bin.write_jar_script libexec/"jenkins.war", "jenkins-lts", :java_version => "1.8"
-    bin.write_jar_script libexec/"cli-#{version}.jar", "jenkins-lts-cli", :java_version => "1.8"
+    (bin/"jenkins-lts").write <<~EOS
+      #!/bin/bash
+      exec "#{Formula["openjdk@11"].opt_bin}/java" $JAVA_OPTS -jar "#{libexec}/jenkins.war" "$@"
+    EOS
+    (bin/"jenkins-lts-cli").write <<~EOS
+      #!/bin/bash
+      exec "#{Formula["openjdk@11"].opt_bin}/java" $JAVA_OPTS -jar "#{libexec}/cli-#{version}.jar" "$@"
+    EOS
```

Comment:
Since https://github.com/Homebrew/brew/pull/7773, `write_jar_script` should work.

Reply:
Can we make an audit for this then?

Reply:
Possibly. I think the preference would be to make it a Rubocop rule so I'll need to see how to actually do that.